src/pangi/domain/guardrails.py

```
@@ -0,0 +1,47 @@
+"""Framework-free trust and guardrail decision values."""
+
+from enum import StrEnum
+
+
+class TrustLevel(StrEnum):
+    """Trust assigned to data at a Pangi boundary."""
+
+    TRUSTED = "trusted"
+    CONDITIONALLY_TRUSTED = "conditionally_trusted"
+    UNTRUSTED = "untrusted"
+    SECRET = "secret"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "SECRET"

(S105)